en/editor/scene/index.md

```diff
@@ -23,9 +23,17 @@ In 2D view, you can move and rotate view of **Scene Editor** by following operat
 - **F** shortcut：focus editor to the selected node.
 
 
-### Coordinate system and grid
+### Coordinate System And Grid
 Grid in the scene is important reference information for us to place the scene elements. For information of the relationship between coordinate system and node attributes like position, please read [Transform](../../concepts/scene/coord.md) section.
 
+### Scene Gizmo
+The Scene Gizmo is in the upper-right corner of the Scene View. It indicates the view direction of editor camera in scene view. You can change view direction quickly by clicking on it.
+
+![Scene Gizmo](images/scene-gizmo.png)
+
+- Click on the six arrows, you can change to the top, down, left, right, front, back view separately.
```

Comment:
- Click on the six arrows, you can change to the top, down, left, right, front, back view separately.
-->
- Click on the six arrows, you can change to the *top*, *down*, *left*, *right*, *front*, *back* views separately.